lib/hacluster.pm

```diff
@@ -150,44 +146,51 @@ sub check_rsc {
     return (!script_run "$crm_mon_cmd 2>/dev/null | grep -q '\\<$rsc\\>'");
 }
 
-# Add the tag for resource configuration
 sub write_tag {
     my $tag     = shift;
     my $rsc_tag = '/tmp/' . get_var('CLUSTER_NAME') . '.rsc';
 
     return (!script_run "echo $tag > $rsc_tag");
 }
 
-# Get the tag for resource configuration
 sub read_tag {
     my $rsc_tag = '/tmp/' . get_var('CLUSTER_NAME') . '.rsc';
 
     return script_output "cat $rsc_tag 2>/dev/null";
 }
 
-# Return the *real* device path
 sub block_device_real_path {
     my $lun = shift;
 
     return script_output "realpath -ePL $lun";
 }
 
-# Add a filter entry in lvm.conf
 sub lvm_add_filter {
     my ($type, $filter) = @_;
     my $lvm_conf = '/etc/lvm/lvm.conf';
 
     assert_script_run "sed -ie '/^[[:blank:]][[:blank:]]*filter/s;\\[[[:blank:]]*;\\[ \"$type|$filter|\", ;' $lvm_conf";
 }
 
-# Remove a filter entry from lvm.conf
```

Comment:
You don't find any of these comments useful or relevant anymore?

Reply:
I changed some function names some PR ago to use more relevant name, so yes these comments are not really useful now.
There is lot of comments in my code, so I try to reduce it a little to only have relevant comments :)